R/utils_render_footnotes.R

```diff
@@ -165,7 +165,8 @@ resolve_footnotes_styles <- function(data,
       ) %>%
       dplyr::mutate(rownum = (rownum / 100) + row_end) %>%
       dplyr::select(-row_start, -row_end) %>%
-      dplyr::mutate(colnum = colname_to_colnum(data = data, colname = colname))
+      dplyr::mutate(colnum = colname_to_colnum(data = data, colname = colname)) %>%
```

Comment:
Let's have `colname_to_colnum` convert NA to 0--if this is only needed sometimes then make it conditional on a required argument